indexer/xml_feature.hpp

```diff
@@ -0,0 +1,47 @@
+#pragma once
+
+#include "geometry/point2d.hpp"
+
+#include "coding/multilang_utf8_string.hpp"
+#include "coding/value_opt_string.hpp"
+
+#include "std/ctime.hpp"
+#include "std/map.hpp"
+#include "std/unique_ptr.hpp"
+
+
+namespace pugi
+{
+class xml_document;
+}
+
+namespace indexer
+{
+
+DECLARE_EXCEPTION(XMLFeatureError, RootException);
+
+class XMLFeature
+{
+public:
+  XMLFeature();
+  XMLFeature(string const & xml);
+
+  pugi::xml_document const & GetXMLDocument() const;
+
+  m2::PointD GetCenter() const;
+
+  string const GetName(string const & lang = "") const;
```

Comment:
String getter version is slower than uint8_t one. It would be better to have 
```
string GetName(string const & lang) const;
string GetName(uint8_t const langCode = StringUtf8Multilang::DEFAULT_CODE) const;
```